src/components/Fields/ContractorPOC/ContractorPOC.Validation.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from "zod";
+
+const finalRule = z
+  .string()
+  .trim()
+  .min(3, "You must select a Contractor POC from the list");
```

Comment:
This is another example of the "3"s we talked about earlier. Not going to flag all of them, but may consider changing it to something of just 1. Main part is I think if you don't put a min, then Zod allows an empty string. So if you want to enforce a value, then you have to set a min. Might be another parameter to make it be a required non-empty string, but think min works just fine.